projects/libtiff/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+RUN apt-get update && apt-get install -y make autoconf automake libtool cmake nasm
+
+RUN git clone --depth 1 https://gitlab.com/libtiff/libtiff
```

Comment:
remove `--depth 1` option here and later. checkout will fail when new commits appear

Reply:
done